### PR TITLE
Feature/griz 291 (allow admins to edit product images)

### PR DESCRIPTION
**Pull request by lawrencecognizant**
This works for the most part, but the edits aren't being saved (because that's back-end work). Once that's implemented, this should work fine.

### CODE REVIEW

**Review by spcognizant — APPROVED** [review]

File: src/components/portal/detailedProduct/ProductDescription.js [hunk 1]
```diff
@@ -81,23 +115,37 @@ class ProductDescription extends Component {
...
-    const product = this.props.product.single;
-    if (!isEmpty(product.images)) {
-      return product.images.map((img, index) => (
+    // if we don't have any images yet, use the incoming product's
+    let images;
+    if (this.files.length === 0) {
+      images = this.props.product.single.images;
+    } else {
+      // console.log(this.pictures);
+      // console.log(this.files);
```
Comment (dansali) [inline src/components/portal/detailedProduct/ProductDescription.js:130]: left over comments for debugging ?

File: src/components/portal/products/ImageUploader.js [hunk 2]
```diff
@@ -187,6 +187,8 @@ class ImageUploader extends Component {
   }
 
   renderPreviewPictures() {
+    // console.log(this.state.pictures);
+    // console.log(this.state.files);
```
Comment (dansali) [inline src/components/portal/products/ImageUploader.js:191]: and here

File: src/components/portal/products/ProductForm.js [hunk 3]
```diff
@@ -39,9 +39,11 @@ class ProductForm extends Component {
...
   }
 
-  onDrop(pictureFiles, pictureDataURLs) {
-    this.pictures = pictureFiles;
-    this.files = pictureDataURLs;
+  onDrop(pictureDataURLs, pictureFiles) {
+    this.pictures = pictureDataURLs;
+    this.files = pictureFiles;
+    // console.log(this.pictures);
+    // console.log(this.files);
```
Comment (dansali) [inline src/components/portal/products/ProductForm.js:46]: and comments also here

**Review by dansali — APPROVED** [review]
You should strip those commented console.logs out.

**Pull request merged**